woocommerce/woocommerce
# Update Action Scheduler to v2.2.3

### All Submissions:


### Changes proposed in this Pull Request:

Update Action Scheduler to restore PHP 5.2 compatibility broken in 2.2.1 / #23016.

I also took the opportunity to remove some development related assets from the bundled version, like the `/tests/` directory, and dotfiles (e.g. `.gitignore` / `.travis.yml`). This makes the changeset look much bigger than it actually is.

Closes #23121.

### How to test the changes in this Pull Request:

1. Checkout the branch
1. Make sure you have a webhook enabled
1. Run through the steps to trigger that webhook
1. Confirm the webhook was sent to the Delivery URL
1. Find the history of the webhook in [Action Scheduler's admin view](https://github.com/Prospress/action-scheduler/#managing-scheduled-actions)

### Other information:




### Changelog entry

> Dev: update Action Scheduler library to version 2.2.3.
